skt/__init__.py

```diff
@@ -406,13 +406,19 @@ def merge_patchwork_patch(self, uri):
 
     def merge_patch_file(self, path):
         if not os.path.isfile(path):
-            raise Exception("Patch %s not found" % os.path.basename(path))
+            raise Exception("Patch %s not found" % os.path.abspath(path))
```

Comment:
Can you please do this change in the commit that introduced the exception? That way if anything happens and the commit needs to be reverted it is easy to do so. We are currently working on an addition to contributing guide (PR #112) that mentions it.

Reply:
Sorry my bad. Let me rebase and do again then

Reply:
This can produce incorrect error message. E.g. when somebody enters a path to a directory instead of a file (e.g. from botched auto-complete). So, instead use `os.path.exists`, and let the "directory instead of file" produce an exception. That will be less confusing.

Reply:
@danghai, could you please address my comment here? I.e. use `os.path.exists`, not `os.path.isfile`.